package.json

```diff
@@ -61,102 +61,102 @@
 		"@fullcalendar/moment-timezone": "^4.4.2",
 		"@fullcalendar/timegrid": "^4.4.2",
 		"@hpi-schul-cloud/commons": "^1.3.3",
-		"@sentry/browser": "^5.6.1",
-		"@sentry/integrations": "^5.6.1",
-		"@sentry/node": "^5.7.1",
-		"acorn": "^6.4.1",
+		"@sentry/browser": "^5.30.0",
+		"@sentry/integrations": "^5.30.0",
+		"@sentry/node": "^5.30.0",
+		"acorn": "^6.4.2",
 		"body-parser": "^1.19.0",
 		"ckeditor5-math": "^20.0.0",
 		"compression": "^1.6.2",
 		"connect-redis": "^4.0.3",
 		"cookie-parser": "~1.4.3",
-		"cross-env": "^7.0.2",
+		"cross-env": "^7.0.3",
 		"csurf": "^1.10.0",
-		"csv-parse": "^4.6.5",
+		"csv-parse": "^4.16.0",
 		"debug": "~2.6.0",
 		"device-detector-js": "^1.1.1",
 		"encoding-japanese": "^1.0.30",
 		"express": "^4.17.1",
-		"express-fileupload": "^1.1.6",
-		"express-prom-bundle": "^6.3.0",
-		"express-session": "^1.16.2",
+		"express-fileupload": "^1.2.1",
+		"express-prom-bundle": "^6.3.6",
+		"express-session": "^1.17.2",
 		"feedr": "^2.13.5",
 		"font-awesome": "^4.7.0",
 		"form-serializer": "^2.5.0",
-		"generate-password": "^1.5.1",
+		"generate-password": "^1.6.1",
 		"git-repo-info": "^2.1.0",
 		"gulp-change": "^1.0.2",
 		"gulp-cli": "^2.3.0",
 		"gulp-rename": "^2.0.0",
-		"handlebars": "^4.7.6",
```

Comment:
what is the dependency which crashed the client? that one should be changed only and it's version should be fixed (to noch change by itself)

Reply:
look into the ticket in the description https://ticketsystem.hpi-schul-cloud.org/browse/BC-8